js/admin/user-form-settings.js

```diff
@@ -158,6 +180,72 @@
       }
     });
 
+    function add_element_to_list(input_id, list_id, hidden_id) {
+      let input = document.getElementById(input_id);
+
+      if (false === input.validity.valid) {
+        return;
+      }
+      if (input.value === '') {
+        return;
+      }
+      let new_value = input.value;
+
+      let hidden_input = document.getElementById(hidden_id);
+      let options_list = hidden_input.value.split(',');
+      if (options_list.includes(new_value)) {
+        input.value = '';
+        return;
+      }
+
+      let list = document.getElementById(list_id);
+      let newOption = new Option(new_value, new_value.toLowerCase());
+      list.add(newOption, undefined);
+
+      input.value = '';
+      if (hidden_input.value === '') {
+        hidden_input.value = new_value;
+      } else {
+        hidden_input.value += `,${new_value}`;
+      }
+      check_change(input);
+    }
+
+    $('.new-field-options').keypress(function(event){
+      let keycode = (event.keyCode ? event.keyCode : event.which);
+      if(keycode == '13'){
+        event.preventDefault();
+
+        let input_option_id = this.id;
+        let list_option_id = input_option_id.replace('input', 'list');
+        let hidden_option_id = input_option_id.replace('input', 'hidden');
+        add_element_to_list(this.id, list_option_id, hidden_option_id);
+      }
+    });
+
+    $('.add-new-field-options').on('click', function() {
+      let add_option_id = this.id;
+      let input_option_id = add_option_id.replace('add', 'input');
+      let list_option_id = add_option_id.replace('add', 'list');
+      let hidden_option_id = add_option_id.replace('add', 'hidden');
+      add_element_to_list(input_option_id, list_option_id, hidden_option_id);
+    });
+
+    $('.list-field-options').on('change', function(event) {
+      let value = event.target.value;
+      $(`#${this.id} option[value='${value}']`).remove();
+      $(`#${this.id}`).val('instruction');
```

Comment:
utilise getelementbyID puis .select popur les sous element

et direct avec this 
```js
var $me = $(this);
$me.find("option[value='${value}']").remove();
$me.val('instruction');
```